drivers/hmis_data_quality_tool/app/models/hmis_data_quality_tool/enrollment.rb

```diff
@@ -166,6 +202,10 @@ def report_item_fields_from_enrollment(report_items:, enrollment:, report:) # ru
       report_item.entry_date = enrollment.EntryDate
       report_item.move_in_date = enrollment.MoveInDate
       report_item.exit_date = enrollment.exit&.ExitDate
+      report_item.entry_date_entered_at = enrollment.DateCreated
+      report_item.exit_date_entered_at = enrollment.exit&.DateCreated
+      report_item.days_to_enter_entry_date = (enrollment.DateCreated.to_date || Date.current) - enrollment.EntryDate
```

Comment:
Maybe return 0 if `DateCreated` is bad? Seems like using `Date.current` will inconsistently surface HMIS export problems instead of DQ?